curriculum/challenges/english/18-project-euler/project-euler-problems-1-to-100/problem-78-coin-partitions.md

```diff
@@ -75,29 +75,34 @@ coinPartitions(7);
 
 ```js
 function coinPartitions(divisor) {
-  const partitions = [1];
+  const partitions = Array(divisor + 1).fill(0);
+  partitions[0] = 1;
+
+  for (let i = 1; i <= divisor; i++) {
```

Comment:
Minor - are there words we can use for what i, j, k mean? People do look at these sometimes so I try to make them clear

Reply:
Hmm, actually:
`i` - is the same as `n` in the initial solution, so it's like `currentN`.
`j` - could be `pentagonalTerm`.
`k` - could be `pentagonalIndex`.